bench/db_bench.ml

```diff
@@ -46,15 +46,29 @@ let populate () =
   in
   loop 0
 
-let print_results db f nb_entries =
+let print_results db f nb_entries operation =
```

Comment:
maybe rename this, it's no longer printing, but it returns a Json. 
also I think we don't need both `json` and `Log.app`. what do you think of letting the user decide which type of output it wants? for instance, if no json file is provided then only print to terminal ?